test/unit_test/model/test_pytorch_model.py

```diff
@@ -33,7 +33,7 @@ def setup(self, tmpdir):
         self.tokenizer = transformers.AutoTokenizer.from_pretrained(self.architecture)
         self.input_text = "Today was an amazing day."
         self.hf_conf = {
-            "task_type": self.task,
+            "task": self.task,
```

Comment:
this is still `task_type` since it is being used by an external function.

Reply:
Oh, True. I do not realize that the mlflow predictor need task_type. Could you help add a few comments in your PR?

Reply:
Anyway, as the CI takes longer time to run. We can add the comments later. Thanks.

Reply:
Already pushed the comment haha